composer/datasets/brats.py

```diff
@@ -264,8 +264,8 @@ def get_split(data, idx):
 def get_data_split(path: str):
     try:
         from sklearn.model_selection import KFold
-    except ImportError as e:
-        raise ImportError(
+    except MissingConditionalImportError as e:
```

Comment:
The `except` should still be an `ImportError`, as the external libraries will not raise `MissingConditionalImportError`.